src/handler-builder.ts

```diff
@@ -305,13 +305,20 @@ export class HandlerBuilder {
 
   /**
    * Create a handler for Cloud Pub/Sub events.
-
-   * `pubsub.onPublish` handles the publication of messages to a topic.
- 
+   * 
+   * `topic.onPublish` handles messages published to a Pub/Sub topic, from SDKs, Cloud Console or Gcloud CLI. 
```

Comment:
If those are three sources of publishing events, then I think omitting the comma after topic, and then adding one after "Cloud Console" will be more clear.

Should the "G" be capitalized in gcloud?

Reply:
Fixed! And you're right, gcloud seems to be lower case in the docs.